linux_os/guide/system/accounts/accounts-restrictions/password_storage/accounts_password_last_change_is_in_past/rule.yml

```diff
@@ -0,0 +1,33 @@
+documentation_complete: true
+
+title: 'Ensure all users last password change date is in the past'
+
+description: |-
+    All users should have a password change date in the past.
+rationale: |-
+    If a users recorded password change date is in the future then they could
+    bypass any set password expiration.
+
+severity: medium
+
+identifiers:
+    cce@sle12: CCE-92330-0
+    cce@sle15: CCE-92504-0
+
+references:
+    cis@sle12: 5.4.1.6
+    cis@sle15: 5.4.1.6
+
```

Comment:
```suggestion
```

Reply:
:+1: changed in [135d663](https://github.com/ComplianceAsCode/content/pull/10031/commits/135d663c3dac8a18fb51088c0b758f095081c5fd)